stable/sonarr/item.yaml

```diff
@@ -1,3 +1,3 @@
 categories:
   - media
-icon_url: https://forums.sonarr.tv/uploads/default/original/2X/e/ef4553fe96f04a298ec502279731579698e96a9b.png
+icon_url: https://github.com/Sonarr/Sonarr/blob/phantom-develop/Logo/512.png?raw=true
```

Comment:
or do we want: 

https://raw.githubusercontent.com/Sonarr/Sonarr/phantom-develop/Logo/512.png

Reply:
@natewalck both should work afaik.